app/models/follow.rb

```diff
@@ -40,7 +40,7 @@ def touch_follower
   def create_chat_channel
```

Comment:
Since this is executed in the `after_create` callback and relies on the follow ID we are going to need to move it to an `after_create_commit` callback to ensure that the record has been committed to the database for the job to find. Checkout [this PR for more details](https://github.com/thepracticaldev/dev.to/pull/5341)

Reply:
yep sure!